### PR TITLE
Add rgba

### PR DESCRIPTION
**Pull request by miniBill**

### CODE REVIEW

File: src/Internal/Style2.elm [hunk 1]
```diff
@@ -48,11 +48,12 @@ compactQuad x y =
 
 
 color : Color -> String
-color (Rgb red green blue) =
+color (Rgb red green blue alpha) =
     "rgb("
         ++ String.fromInt red
         ++ ("," ++ String.fromInt green)
         ++ ("," ++ String.fromInt blue)
+        ++ ("/" ++ String.fromFloat alpha)
```
Comment (lydell) [inline src/Internal/Style2.elm]: Would it make sense to use the `rgba()` CSS function instead for better browser support?

Comment (miniBill) [reply src/Internal/Style2.elm]: Maybe? I can't find data on support for `rgba` vs `rgb( / alpha)`

Comment (lydell) [reply src/Internal/Style2.elm]: The alpha parameter of `rgb` is fairly new: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#browser_compatibility

While `rgba` is ancient: https://css-tricks.com/rgba-browser-support/

Edit: Here’s a better table: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#browser_compatibility

Comment (miniBill) [reply src/Internal/Style2.elm]: Done!

**Comment by miniBill** [comment]
Not needed anymore!

**Pull request closed without merging**